plugins/processors/filepath/filepath.go

```diff
@@ -82,6 +82,7 @@ func (o *Options) processMetric(metric telegraf.Metric) {
 	// Rel
 	for _, v := range o.Rel {
 		o.applyFunc(v.BaseOpts, func(s string) string {
+			//nolint:errcheck // no way to propagate the error
 			relPath, _ := filepath.Rel(v.BasePath, s)
 			return relPath
```

Comment:
I would at least like to see a log message for the error and returning the initial path instead of the (potentially) invalid/empty return value. What do you think?